samcli/commands/init/command.py

```diff
@@ -238,12 +218,12 @@ def wrapped(*args, **kwargs):
 @click.option(
     "--tracing/--no-tracing",
     default=None,
-    help="Enable AWS X-Ray tracing for your lambda functions",
+    help="Enable AWS X-Ray tracing for application.",
 )
 @click.option(
     "--application-insights/--no-application-insights",
     default=None,
-    help="Enable CloudWatch Application Insights monitoring for your application",
+    help="Enable CloudWatch Application Insights monitoring for application.",
```

Comment:
nit: I remember seeing another PR which removed the trailing dots in a help text, we might need to align which approach we are taking

Reply:
Oh interesting, do we remember the rationale for that? Agree on adopting one vs other in totality for consistency.

Reply:
I think that is more of a personal preference, not sure if CLI recommendations favor one or the other. I am OK with either but just wanted to make sure we will have a persistent help text through the CLI. :)